src/pepr/operator/controllers/keycloak/client-sync.ts

```diff
@@ -188,7 +184,7 @@ export function handleClientGroups(clientReq: Sso) {
   delete clientReq.groups;
 }
 
-async function apiCall(sso: Partial<Sso>, method = "POST", authToken = "") {
+export async function apiCall(sso: Partial<Sso>, method = "POST", authToken = "") {
```

Comment:
I think with the change to not need the GET we don't need this exported anymore?